handlers/deploy.go

```diff
@@ -156,6 +157,16 @@ func makeDeploymentSpec(request requests.CreateFunctionRequest, existingSecrets
 		return nil, resourceErr
 	}
 
+	var imagePullPolicy apiv1.PullPolicy
+	switch config.ImagePullPolicy {
+	case "Never":
```

Comment:
Just curious if if the value of `apiv1.PullNever` is the same as the string "Never" or whether it is an int?

Reply:
It ends up aligning with string constants in the k8s api, but I preferred to stick to the native types being used to construct the deploy object.